cmd/cone/search_entitlements.go

```diff
@@ -98,6 +98,18 @@ func (r *ExpandedEntitlementsResponse) Rows() [][]string {
 	}
 	return rows
 }
+func (r *ExpandedEntitlementsResponse) sortByColumn() string {
+	return "Display Name"
```

Comment:
Can you make this (and the header in the `Header() []string` code) a constant, so if you change one it changes the other

Also, this func should be `SortByColumnName() string`